### PR TITLE
docs(readme): Add section for signing commits and tags

### LINKED ISSUE #94: [question] Tags are not signed with GPG key
As mentioned in gitter, I have found the tags I've been creating with `standard-version` are not being signed with my GPG key:

screenshot 2016-08-22 20 05 24

### PR DESCRIPTION
**Pull request by JaKXz**
Resolves #94.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,9 @@ npm run release -- --no-verify
 standard-version --no-verify
 ```
 
+### Signing commits and tags
```
Comment (bcoe) [inline README.md]: sorry for the nit, but mind adding a space below the heading, I think this might cause issues for some markdown renderers.

**Comment by JaKXz** [comment]
@bcoe fixed :)

**Comment by stevemao** [comment]
Thanks @JaKXz ! I thought the docs was updated. LGTM

**Comment by Tapppi** [comment]
Thanks @JaKXz!

**Pull request merged**